pkg/names/names.go

```diff
@@ -35,6 +35,11 @@ const MULTUS_NAMESPACE = "openshift-multus"
 // ALLOWLIST_CONFIG_NAME is the name of the allowlist ConfigMap
 const ALLOWLIST_CONFIG_NAME = "cni-sysctl-allowlist"
 
+// CRON_SCHEDULER_CONFIG_NAME is the name of the cronjob schedule configmap
+const CRON_SCHEDULER_CONFIG_NAME = "cron-scheduler-configmap"
+
+const USER_EDITABLE_CRON_SCHEDULER_CONFIGMAP = "cron-scheduler-configmap"
```

Comment:
If these values are the same, do we need both?

Reply:
We don't, good catch.